providers/launchdarkly/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/open-feature/go-sdk-contrib/providers/launchdarkly
+
+go 1.20
```

Comment:
This needs to be adjusted to comply with LD supported version - which is 1.18 [1][2]

[1] - https://github.com/launchdarkly/go-server-sdk#supported-go-versions
[2] - https://github.com/launchdarkly/go-sdk-common#supported-go-versions